todo_app/data/session_items.py

```diff
@@ -1,8 +1,8 @@
 from flask import session
 
 _DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
+    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items', 'completed': False },
```

Comment:
I think I'd argue we don't need both the status and the completed flag; we could probably just remove one of those (but having said that, knowing what M2 involves I'll probably say don't bother here - it will change anyway!)